### ISSUE TITLE
Pagination does not work

### ISSUE THREAD

**Issue by ovninosa** [LiveControl/EloquentDataTable#3]
I wrote a simple code to use the library but the pagination does not work.

The used list on the query has 13 entries and the query result from the EloquentDataTable always return the 13 entries even in the page 1 or 2 of the datatables.

The Datatables used version:
//cdn.datatables.net/1.10.6/js/jquery.dataTables.min.js

The route:

``` php
Route::get('home/datatable', 'HomeController@datatable');
```

The controller code:

``` php
    public function datatable()
    {
        $users = new User();
        $dataTable = new DataTable(
            $users,
            ['email', 'name', 'password']
        );

        $dataTable->setFormatRowFunction(function ($user) {
            $row = [];
            $row[] = '<a href="/users/'.$user->id.'">'.$user->name.'</a>';
            $row[] = '<a href="mailto:'.$user->email.'">'.$user->email.'</a>';
            $row[] = '<a href="/users/delete/'.$user->id.'">&times;</a>';
            return $row;
        });

        return response()->json($dataTable->make());
    }
```

The HTML code:

``` html
<table id="users" class="table table-hover table-condensed">
    <thead>
        <tr>
            <th class="col-md-3">{{{ Lang::get('users/table.username') }}}</th>
            <th class="col-md-3">{{{ Lang::get('users/table.email') }}}</th>
            <th class="col-md-3">{{{ Lang::get('users/table.created_at') }}}</th>
        </tr>
    </thead>
</table>
```

The JS code:

``` javascript
$(document).ready(function() {
    var table = $('#users').DataTable({
        "processing": true,
        "serverSide": true,
        "ajax": "/home/datatable"
    });
});
```

**Comment by jeffreydevreede** [comment]
This is because the plugin (currently) only works with POST values. If you check out the latest README.md on master-dev you can see the following example for the js code:

``` javascript
var table = $('#example').DataTable({
  "processing": true,
  "serverSide": true,
  "ajax": {
    "url": "<url to datatable route>",
    "type": "POST"
  }
});
```

So in your case this should become:

``` javascript
var table = $('#users').DataTable({
  "processing": true,
  "serverSide": true,
  "ajax": {
    "url": "/home/datatable",
    "type": "POST"
  }
});
```

**Comment by ovninosa** [comment]
Jeffrey, when I use the method as POST.

The server throw an error about the csrf token, do you have any idea to fix this?

Thanks in advance.

**Comment by jeffreydevreede** [comment]
I would recommend you to take a look at http://laravel.com/docs/master/routing#csrf-protection especially that part about X-CSRF-TOKEN.

If you can't get that working, and assuming you are using inline script in a blade view, this problem can also be fixed as following:

``` javascript
var table = $('#example').DataTable({
  "processing": true,
  "serverSide": true,
  "ajax": {
    "url": "<url to datatable route>",
    "type": "POST",
    "data": function (data) {
      data._token = "{{{ csrf_token() }}}";
    }
  }
});
```

**Comment by ovninosa** [comment]
If I use that block of javascript code the csrf works fine.
But the query by eloquent throw this exception:

SQLSTATE[42803]: Grouping error: 7 ERROR: column «users.email» must appear in the GROUP BY clause or be used in an aggregate function
LINE 1: ...elect count(_) as aggregate from "users" order by "email" as...
^ (SQL: select count(_) as aggregate from "users" order by "email" asc)

The user model is the default provided by Laravel 5. And the DB Driver is for postgres.
The entire code is the same as this thread with your JS to avoid CSRF.
Do you have any idea of this?

**Comment by jeffreydevreede** [comment]
Fixed this bug in issue #3. Please do a composer update to get the latest version.
